viapy/api.py

```diff
@@ -198,22 +216,22 @@ class SRUItem(AttrMap):
     @property
     def uri(self):
         """VIAF URI for this result"""
-        return self.recordData.Document["@about"]
+        return self.recordData.VIAFCluster.Document["about"]
```

Comment:
Do we need to preserve this `VIAFCluster` layer of nesting? Can we update so that recordData == VIAFCluster and everythings nests under it as it did before? I was trying to follow in the code where that got added or changed, and it wasn't obvious (I guess it's due to the API changes).

If that doesn't work, at least we need a little bit more content in the comments explaining the change.

Reply:
Ah, my mistake - if this is only for the `SRUItem` class, I don't care about the change as much. I'm fine with the change, but a brief comment on the change would be helpful. (No need to repeat, could just add a comment near thet op that the viafcluster was added in a 2025 update to match changes in the API or something like that)